internal/vpn/client.go

```diff
@@ -248,17 +267,25 @@ func (c *Client) setupDirectRoute(ip net.IP) error {
 	return nil
 }
 
+func (c *Client) removeDirectRoute(ip net.IP) error {
+	if !ip.IsLoopback() {
+		c.log.Infof("Removing direct route to %s", ip.String())
+		if err := DeleteRoute(ip.String()+directRouteNetmaskCIDR, c.defaultGateway.String()); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
 func (c *Client) removeDirectRoutes() {
 	c.directIPSMu.Lock()
 	defer c.directIPSMu.Unlock()
 
 	for _, ip := range c.directIPs {
-		if !ip.IsLoopback() {
-			c.log.Infof("Removing direct route to %s", ip.String())
-			if err := DeleteRoute(ip.String()+directRouteNetmaskCIDR, c.defaultGateway.String()); err != nil {
-				// shouldn't return, just keep on trying the other IPs
-				c.log.WithError(err).Errorf("Error removing direct route to %s", ip.String())
-			}
+		if err := c.removeDirectRoute(ip); err != nil {
+			// shouldn't return, just keep on trying the other IPs
+			c.log.WithError(err).Errorf("Error removing direct route to %s", ip.String())
```

Comment:
What do you think about changing the logging level to warning here as no error is returned?